sys/kern/sys_process.c

```diff
@@ -663,9 +663,11 @@ sys_ptrace(struct thread *td, struct ptrace_args *uap)
 		syscallarg_t args[nitems(td->td_sa.args)];
 		struct ptrace_sc_ret psr;
 		int ptevents;
-	} r;
-	void * __capability addr;
-	int error = 0;
+	} r = { 0 };
```

Comment:
Why has all this changed? We don't need to zero-initialise, and we can leave the `addr = &r` being lower down. Please keep your diffs minimal and don't just reorganise things because it makes it harder to see what's actually being changed and what's just moved for the sake of it.